### PR TITLE
Add Api musicapi

### PR DESCRIPTION
**Pull request by cagin24**

### CODE REVIEW

**Comment by GozdeUnver** [comment]
pycache and .DS_Store files must be removed

**Comment by GozdeUnver** [comment]
Unit tests must be written and pushed today

**Comment by GozdeUnver** [comment]
I couldn't see any post methods for the api. I only see 2 html rendered functions for the UI of the api and 1 get method that returns a json response.

File: practice-app/group5_api/musicapi/views.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        url = "https://api.spotify.com/v1/playlists/{}/tracks".format(settings.PLAYLIST_ID)
+        headers = settings.TOKEN
+        r = requests.get(url, headers=headers)
+        info = r.content
+        json_response = r.json()
+        repository = json_response['items']
+        list = []
+        for i in repository:
+            list.append(i['track']['name'])
+        return Response(list)
```
Comment (GozdeUnver) [inline practice-app/group5_api/musicapi/views.py:30]: Must add a function to handle post requests to the API

**Comment by GozdeUnver** [comment]
API documentation should also be added

**Comment by cagin24** [comment]
> I couldn't see any post methods for the api. I only see 2 html rendered functions for the UI of the api and 1 get method that returns a json response.

T.A stated that GET method is enough for this task.

File: practice-app/group5_api/musicapi/tests.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+from rest_framework import status
+from rest_framework.test import APITestCase, APIClient
 
-# Create your tests here.
+class musicapi(APITestCase):
+
+    # Test GET method
+    def test_get_method(self):
+        client = APIClient()
+        response = client.get("/musicapi/api/")
```
Comment (GozdeUnver) [inline practice-app/group5_api/musicapi/tests.py:10]: This uri is not present in the urls.py of this module. Paths must be updated

**Comment by GozdeUnver** [comment]
It looks fine you can merge it

**Comment by GozdeUnver** [comment]
It looks fine you can merge it

**Comment by GozdeUnver** [comment]
Now I realized that you never used "info = r.content" in class GetMusicView(APIView) @cagin24 , is that why your api is not working? You said it was because of the time limitation of the api but I guess that may not be the case. Could you explain why you didn't use r.content

**Pull request merged**